src/utils/compare-bytecode.ts

```diff
@@ -1,5 +1,5 @@
 
-// TODO dep: this does not work. find a better way and uncomment the call in meow-token.ts
+// TODO dep: this does not work. find a better way and uncomment the call in z-token.ts
 const replacePattern = /a165627a7a72305820.{64}0029.*/g;
```

Comment:
Since this code is pending more work in a different area it would be nicer to remove this and just make a new branch for this work specifically.